hdInsight-apache-spark-1-4-1/azuredeploy.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
```

Comment:
We're going to start testing an automated testing bot for PRs soon. It'd be great to have valid values on these parameters below. For parameter such as clusterName that requires a unique value, you can put in "#####" and we'll auto-gen a random lower cased alphabetical string for it.